rotkehlchen/globaldb/handler.py

```diff
@@ -462,6 +462,8 @@ def get_assets_mappings(identifiers: list[str]) -> tuple[dict[str, dict], dict[s
                             'name': entry[7],
                             'symbol': entry[8],
                         }
+                if entry[9] is not None and entry[9] == SPAM_PROTOCOL:
```

Comment:
Why do you need the extra comparison?

```suggestion
                if entry[9] == SPAM_PROTOCOL:
```